src/framework/handlers/shader.js

```diff
@@ -32,6 +40,10 @@ class ShaderHandler {
         });
     }
 
+    openBinary(data) {
```

Comment:
Please add documentation to `handler.js` on what this function is, when it is used, what the data parameter is etc.

Reply:
Added individual functions to parsers, as not every handler can/should support openBinary method.